queries.sql

```diff
@@ -0,0 +1,141 @@
+--
+--
+--RESETS
+DROP TABLE IF EXISTS organization;
+DROP TABLE IF EXISTS channel;
+DROP TABLE IF EXISTS user;
+DROP TABLE IF EXISTS message;
+DROP TABLE IF EXISTS channel_user;
+-- RESETS
+--
+--
+
+--
+--
+-- CREATES
+CREATE TABLE organization (
+  id INTEGER PRIMARY KEY AUTOINCREMENT,
+  name VARCHAR(128) NOT NULL
+);
+
+CREATE TABLE channel (
+  id INTEGER PRIMARY KEY AUTOINCREMENT,
+  name VARCHAR(128) NOT NULL,
+  organization_id INTEGER REFERENCES organization(id)
+);
+
+CREATE TABLE user (
+  id INTEGER PRIMARY KEY AUTOINCREMENT,
+  name VARCHAR(128) NOT NULL
+);
+
+CREATE TABLE message (
+  id INTEGER PRIMARY KEY AUTOINCREMENT,
+  content VARCHAR(128) NOT NULL,
+  post_time VARCHAR(128) NOT NULL,
+  user_id INTEGER REFERENCES user(id),
+  channel_id INTEGER REFERENCES channel(id)
+);
+
+CREATE TABLE channel_user (
+  channel_id INTEGER REFERENCES channel(id),
+  user_id INTEGER REFERENCES user(id)
+);
+-- CREATES
+--
+--
+
+--
+--
+-- INSERTS
+INSERT INTO organization (name) VALUES ("Lambda School");
+
+INSERT INTO user (name) VALUES ("Alice");
+INSERT INTO user (name) VALUES ("Bob");

```

Comment:
It is also possible to write insert query in one line by inserting multiple values with coma.